spec/proforma/importer_spec.rb

```diff
@@ -125,5 +141,44 @@
 
       it { is_expected.to be_an_equal_task_as ref_task }
     end
+
+    context 'with a specific expected_version' do
+      let(:importer) { described_class.new(zip_file, expected_version) }
+      let(:expected_version) { '2.0' }
+
+      context 'when exported_version is the same' do
+        let(:export_version) { expected_version }
+
+        it 'does not raise an error' do
+          expect { perform }.not_to raise_error
+        end
+      end
+
+      context 'when exported_version is different' do
```

Comment:
I didn't get that test (and the corresponding importer). Why could the exported_version be different? From what?

Reply:
I wanted to be able to restrict the version used for the import. This spec should check for that.

Reply:
Ah, okay. Thanks